docs/guide/migration/index.md

```diff
@@ -67,6 +67,18 @@ createRouter({
 })
 ```
 
+### Removal of the `fallback` option
+
+The `fallback` option is no longer supported when creating the router:
+
+```diff
+-new VueRouter({
++createRouter({
+-  fallback: false,
+// other options...
+})
+```
+
```

Comment:
```suggestion

**Reason**: All browser supported by Vue support the [HTML5 History API](https://developer.mozilla.org/en-US/docs/Web/API/History_API), allowing us to avoid hacks around modifying `location.hash` and directly use `history.pushState()`.

```

Reply:
Sorry, I didn't see your comment and the suggestion was messed up in GitHub review

Reply:
I pushed a new commit with this suggestion modified. :-)